src/CVRF/cvrf_priv.c

```diff
@@ -140,7 +140,7 @@ struct oscap_string_iterator *cvrf_score_set_get_product_ids(struct cvrf_score_s
 
 bool cvrf_score_set_add_metric(struct cvrf_score_set *score_set, enum cvss_category category, const char *score) {
 	struct cvss_metrics *metric = cvss_metrics_new(category);
-	cvss_metrics_set_score(metric, oscap_strtol(score, NULL, 10));
+	cvss_metrics_set_score(metric, strtod(score, NULL));
```

Comment:
"Abc" would be an invalid score. It has to be a float according to CVRF specifications